docs/usage.md

```diff
@@ -0,0 +1,40 @@
+# Using Config Sync
+
+This document provides instructions on how to use Config Sync in a kubernetes
```

Comment:
nit: capitalize Kubernetes following the K8s doc style guide https://kubernetes.io/docs/contribute/style/style-guide/#kubernetes-io-word-list